scripts/kind-e2e/e2e.sh

```diff
@@ -269,10 +285,6 @@ for i in 2 3; do
     verify_subm_operator_container
 
     # FIXME: Rename all of these submariner-engine or engine, vs submariner
-    # Create SubM CR
```

Comment:
We can do it in a follow-up patch, but we shouldn't forget to get rid of these functions.

Reply:
Done.